sphinx_gallery/clean.py

```diff
@@ -0,0 +1,29 @@
+from argparse import ArgumentParser
+import os.path
+import shutil
+import subprocess
+import sys
+
+
+def main():
+    parser = ArgumentParser(
+        description=
+        "Clean intermediate files generated by sphinx-gallery, and call "
+        "'python -msphinx -M clean' with the same arguments.")
+    parser.add_argument("sourcedir")
```

Comment:
Very ignorant question but which are the defaults?

Reply:
It's a required argument, so there's no default.